src/applications/vaos/appointment-list/components/ScheduleNewAppointmentRadioButtons.jsx

```diff
@@ -26,19 +26,22 @@ import NewTabAnchor from '../../components/NewTabAnchor';
  * />
  * @module appointment-list/components
  */
-function ScheduleNewAppointmentRadioButtons({
-  canUseVaccineFlow,
-  directScheduleSettingsStatus,
-  fetchDirectScheduleSettings,
-  startNewAppointmentFlow,
-  startNewVaccineFlow,
-}) {
+export default function ScheduleNewAppointmentRadioButtons() {
   const history = useHistory();
+  const dispatch = useDispatch();
+  const canUseVaccineFlow = useSelector(
+    state => selectCanUseVaccineFlow(state),
+    shallowEqual,
+  );
+  const directScheduleSettingsStatus = useSelector(
+    state => selectDirectScheduleSettingsStatus(state),
+    shallowEqual,
+  );
 
   const [radioSelection, setRadioSelection] = useState();
   useEffect(() => {
     if (directScheduleSettingsStatus === FETCH_STATUS.notStarted) {
-      fetchDirectScheduleSettings();
+      dispatch(actions.fetchDirectScheduleSettings());
```

Comment:
So far we've been going back to destructuring the actions in the import statement, like this: 

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/appointment-list/components/CanceledAppointmentsList.jsx#L6-L9

Reply:
Done